src/dialog/Dialog.ts

```diff
@@ -78,11 +96,15 @@ export default class Dialog extends DialogBase<DialogProperties> {
 			underlay
 		} = this.properties;
 
+		document.body.onkeyup = this._onKeyUp.bind(this);
```

Comment:
Is this necessary, since you already have the `document.addEventListener`?

Reply:
unlikely, must have been a mistake.